src/backend/lib/config.js

```diff
@@ -1,11 +1,10 @@
 const path = require('path');
 const dotenv = require('dotenv');
-const { logger } = require('../utils/logger');
 
 const result = dotenv.config();
 
-if (result.error && logger) {
-  logger.error(
+if (result.error) {
+  console.error(
```

Comment:
`console.warn` vs. `console.error` here please, so we don't include a stack

Reply:
thanks for the advice!